README.md

```diff
@@ -27,34 +27,36 @@ Transform and optimize assets. Visit our documentation to learn more about [medi
 | 0.0.1       | > 1.0           | 
 
 ## Installation
-To use this SDK, add cloudinary as a [dependency in your pubspec.yaml file](https://flutter.dev/platform-plugins/).
+To use this SDK, add the Flutter and Dart Cloudinary libraries as [dependency in your pubspec.yaml file](https://flutter.dev/platform-plugins/).
 
 ```yaml
 dependencies:
   cloudinary_flutter: ^0.0.1
+  cloudinary_dart: ^0.0.6
 ```
 
 ## Usage
 ### Setup
-When setting up your Flutter app, you’ll import the Dart library that was installed as part of your Flutter installation.
 The `Cloudinary` class is the main entry point for using the library. Your `cloud_name` is required to create an instance of this class. Your `api_key` and `api_secret` are also needed to perform secure API calls to Cloudinary (e.g., image and video uploads). Setting the configuration parameters can be done either programmatically using an appropriate constructor of the Cloudinary class or globally using an environment variable. You can find your account-specific configuration parameters in the **Dashboard** page of your [account console](https://cloudinary.com/console).
 
 Here’s an example of setting configuration parameters in your Flutter application:
 
 ```dart
-import 'package:cloudinary_dart/src/cloudinary.dart';
-
-var cloudinary = Cloudinary.withStringUrl('cloudinary://<your-api-key>:<your-api-secret>@<your-cloud-name>');
+CloudinaryContext.cloudinary = Cloudinary.fromCloudName(cloudName: 'demo');
```

Comment:
Will I be able to use this once I bump to the new version? I wasn't able to set config params this way when I tried it now.

Reply:
You'll have to upgrade `cloudinary_dart` to 0.0.7